enhancements/api-review/custom-tags-aws.md

```diff
@@ -73,71 +95,377 @@ Note existing unchanged behavior: The installer will apply these tags to all AWS
 All operators that create AWS resources (ingress, cloud credential, storage, image registry, machine-api)
 will apply these tags to all AWS resources they create.
 
-userTags that are specified in the infrastructure resource will merge with userTags specified in an individual component. In the case where a userTag is specified in the infrastructure resource and there is a tag with the same key specified in an individual component, the value from the individual component will have precedence and be used.
+userTags that are specified in the infrastructure resource will merge with userTags specified in a kube resource. In the case where a userTag is specified in the infrastructure resource and there is a tag with the same key specified in a kube resource, the value from the kube resource will have precedence and be used.
 
 The userTags field is intended to be set at install time and is considered immutable. Components that respect this field must only ever add tags that they retrieve from this field to cloud resources, they must never remove tags from the existing underlying cloud resource even if the tags are removed from this field(despite it being immutable).
 
 If the userTags field is changed post-install, there is no guarantee about how an in-cluster operator will respond to the change. Some operators may reconcile the change and change tags on the AWS resource. Some operators may ignore the change. However, if tags are removed from userTags, the tag will not be removed from the AWS resource.
 
+#### Existing drawbacks
+
+If a customer decides they do not want these tags applied to new resources, there is no clean way to address that need:
+
+1. they would have to get help from support to edit the status field
+2. they would have to manually remove the undesired tags from any existing resources
+
+In the future we will want to introduce a spec field where a customer can specify and edit these tags, which will be reflected into status when changed, and we will expect consuming components to reconcile changes to the set of tags by applying net new tags to existing resources (but they will still not remove tags that are dropped from the list of tags).
+
+### New proposal to support update of user-defined tags (or userTags)
+
+- `experimentalPropagateUserTags` field will be set for deprecation in this proposal version.
+
+  The install validation will fail if there is any tag that starts with `kubernetes.io` or `openshift.io` in `userTags`. In the existing design, kubernetes.io and openshift.io namespaces are not allowed as part of user-defined tags.
+
+  The `resourceTags` field will be populated by the installer using the entries from `userTags` field.
```

Comment:
This should be clear that the installer will only populate the field in the spec.

Reply:
Ok, added the change to EP.